ddp-workspace/projects/ddp-testboston/src/app/components/welcome/welcome.component.scss

```diff
@@ -17,8 +17,8 @@
 
 .anchor {
     display: block;
-    padding-top: 6rem;
-    margin-top: -6rem;
+    padding-top: 3rem;
+    margin-top: -3rem;
```

Comment:
Less space at the top when we scrolled to an anchor.

Reply:
Better!!!